x-pack/plugins/cases/kibana.json

```diff
@@ -1,18 +1,31 @@
 {
-  "configPath": ["xpack", "cases"],
-  "id": "cases",
-  "kibanaVersion": "kibana",
-  "extraPublicDirs": ["common"],
-  "requiredPlugins": [
-    "actions",
-    "esUiShared",
-    "features",
-    "kibanaReact",
-    "kibanaUtils",
-    "triggersActionsUi"
+  "configPath":[
```

Comment:
@cnasikas is there any reason why we changed config from `xpack.cases` to `cases.xpack`? Noticed that since it broke my 7.13 -> 7.14 upgrade ....

Reply:
This is surely a bug. It should not be changed. I will open a PR to fix it for 7.14.1. Sorry for the inconvenience.

Reply:
Could you please tell me what kind of configuration you have for cases?

Reply:
Fixed by https://github.com/elastic/kibana/pull/107637

Reply:
> Could you please tell me what kind of configuration you have for cases?

I just disabled it `xpack.cases.enabled: false` along with a bunch of other plugins in 7.13 :slightly_smiling_face:

Having said that, in 7.14 `cases` became a required dependency of the `observability`, so I re-enabled it anyway. From time to time we recommend customers to disable certain plugins (e.g. if vulnerability is discovered in a plugin and the only way to remediate it before the patch release is available is to disable it), but it's not something we "advertise" for sure.

> Fixed by #107637. @azasypkin Thanks for cathing that.

Awesome :+1:

Reply:
Just wanted to cross post the work around here as well just in case someone needs it in the future: `cases.xpack.enabled: <true|false>` should get kibana running again if you need to specify the option before upgrading 👍